templates/zerver/emails/missed_message.txt

```diff
@@ -12,6 +12,10 @@
 {% endfor %}
 {% endif %}
 
+{% if mention %}
+You are receiving this email because you were mentioned.
```

Comment:
I think we also want to say "You are receiving this email because you configured notifications for this stream" or similar.

Reply:
(But this can be in a later commit, because it's not critical in the way the original change is).